config/admin_ranks.txt

```diff
@@ -30,14 +30,28 @@
 # +EVERYTHING (or +HOST or +ALL) = Simply gives you everything without having to type every flag
 # END_KEYWORDS
 
-Admin Observer
-Moderator		= +ADMIN
-Admin Candidate	= +@
-Trial Admin		= +@ +SPAWN +VAREDIT +BAN
-Badmin			= +@ +POSSESS +POLL +BUILDMODE +SERVER +FUN
-Game Admin		= +@ +STEALTH +SOUNDS +DEBUG
-Game Master		= +EVERYTHING
+## Trial Ranks
+Admin Observer 	    = +@
+Admin Candidate		= +@
 
-Host			= +EVERYTHING
 
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL
+## Junior Ranks
+TrialModerator		= +@ +BAN
+Moderator			= +@ +BAN
+
+
+## Admin Ranks
+Administrator		= +@ +STEALTH +DEBUG +SPAWN +VAREDIT +BAN
+Primarymin			= +@ +STEALTH +DEBUG +SPAWN +VAREDIT +POSSESS +POLL +FUN +BAN
```

Comment:
I thought we agreed to remove primarymin?

Reply:
keep the tg ranks, they make sense

Reply:
We're familiar with a different ranking system. Besides, we've already established our ramp structure. We're not TG.